src/rgw/rgw_bucket.cc

```diff
@@ -3363,7 +3361,7 @@ int RGWBucketCtl::do_link_bucket(RGWSI_Bucket_EP_Ctx& ctx,
     } else {
       ret = svc.bucket->read_bucket_entrypoint_info(ctx,
                                                     meta_key,
-                                                    &ep, &ot,
+                                                    &ep, &pinfo->ep_objv,
```

Comment:
this is in the else block of `if (pinfo) { ... } else {`, so `pinfo` here is a nullptr

Reply:
`pinfo->ep_objv` is also used unchecked later in the call to `put_bucket_entrypoint_info()`